src/ludamus/gates/cli/django/management/commands/downloadvendor.py

```
@@ -31,7 +31,9 @@ class Command(BaseCommand):
 
     help = "Download vendor dependencies to static/vendor/ with SHA-384 verification"
 
-    def add_arguments(self, parser: ArgumentParser) -> None:  # ruff:ignore[no-self-use]
+    def add_arguments(  # ruff: ignore [no-self-use]
```

Ruff (0.16.1)

[warning] Invalid rule code in suppression: no-self-use

Enable `lint.preview` to use rule names

(RUF102)